.github/workflows/commit.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Check Commit Messages'
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+permissions:
+  contents: read
+
+jobs:
+  check-commit-message:
+    name: Check Commit Messages
```

Comment:
Can we think of something smarter here so that it doesn't result in `Check Commit Messages / Check Commit Messages` in the GitHub UI due to `.name` matching `.jobs[].name`?

Reply:
Changed the redundant terms to `PR Validation > Check Commit Messages` instead. PTAL